code/game/objects/effects/spawners/themed_loot_tables.dm

```diff
@@ -179,7 +179,7 @@
 		/obj/item/gun/ballistic/rifle/hunting = 20,
 		/obj/item/gun/ballistic/revolver/widowmaker = 17,
 		/obj/item/gun/ballistic/shotgun/hunting = 10,
-		/obj/effect/spawner/bundle/f13/mosin = 8,
+		//obj/effect/spawner/bundle/f13/mosin = 8,
```

Comment:
Please replace this with a hunting rifle or something.

Reply:
That, or the pipe gun, or some kind of other gun, dont like just removing things.

Reply:
Resolved, replaced with Hunting rifle as requested.